packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -682,17 +690,23 @@ export default class TradeStore extends BaseStore {
         if (!proposal_info) {
             return;
         }
-        const { contract_type, barrier, high_barrier, low_barrier } = proposal_info;
+        const { barrier, contract_type, high_barrier, low_barrier } = proposal_info;
 
         if (isBarrierSupported(contract_type)) {
             const color = this.root_store.ui.is_dark_mode_on ? BARRIER_COLORS.DARK_GRAY : BARRIER_COLORS.GRAY;
-            const turbos_color = getTurbosColor(contract_type);
+            const turbos_color = this.hovered_barrier ? getTurbosColor(contract_type) : BARRIER_COLORS.GRAY;
 
             // create barrier only when it's available in response
-            this.main_barrier = new ChartBarrierStore(barrier || high_barrier, low_barrier, this.onChartBarrierChange, {
-                color: isTurbosContract(contract_type) ? turbos_color : color,
-                not_draggable: isTurbosContract(contract_type),
-            });
+            this.main_barrier = new ChartBarrierStore(
+                this.hovered_barrier || barrier || high_barrier,
+                low_barrier,
+                this.onChartBarrierChange,
+                {
+                    color: isTurbosContract(contract_type) ? turbos_color : color,
```

Comment:
Maybe it'll be better to write color that way: 
`color: this.hovered_barrier ? turbos_color : color` (line 705)
Because only Turbos has `this.hovered_barrier` + we can delete line 697.

Reply:
@kate-deriv the barrier needs to be red for Short. it doesn't matter if it's hovered or not )

Reply:
May we have a call? Because in [Figma](https://www.figma.com/file/xqAQd87EYFFwIqPVKYbeYh/Turbos?node-id=6054%3A606745&t=BKHRlEg4LzNpSKXv-0) barrier for Short is red only in case if it was hovered.

Reply:
Conclusion after call: all barriers are **gray and solid** by default. When user hover, **Long become green + dashed and Short become Red + dashed**.